modules/install/src/main/scala/coursier/install/GraalvmParams.scala

```diff
@@ -2,7 +2,30 @@ package coursier.install
 
 import dataclass.data
 
+import java.util.Locale
+
 @data class GraalvmParams(
-  defaultVersion: Option[String],
+  defaultVersion: String,
   extraNativeImageOptions: Seq[String]
 )
+
+object GraalvmParams {
+  lazy val defaultGraalvmVersion: String = {
+    val defaultVersion = "22.3.0"
```

Comment:
Maybe we could just use the same version [as the coursier build](https://github.com/coursier/coursier/blob/44e1a024897fb1046303f705e0e4143d808e1adc/project/deps.sc#L74) (it would need to be passed there via [some kind of constants file](https://github.com/coursier/coursier/blob/44e1a024897fb1046303f705e0e4143d808e1adc/project/modules/core0.sc#L15-L29))